app/scripts/service/authorized-backend.service.js

```diff
@@ -91,11 +94,17 @@ define([
                       //console.log(UserService.getParsedToken());
                       console.log("Execute original call once again");
                       owner.getHttpPromise(httpConfigObject).then(function (response) {
-                        thenFunction(response);
+                        if (thenFunction) {
```

Comment:
left thenFunction integration intact for now so as not to break the current integrations with this call